### DISCUSSION TITLE
我写了个 SaltyQQChat，依赖 QQAPI，我要不要也使用 GPL 3.0？

### DISCUSSION THREAD

**Discussion by SALTWOOD** [AnzhiZhang/MCDReforgedPlugins#254] [Q&A] [answered]
没包含源代码，问了 AI 说是可以不用，但也不知道对不对所以来问问（
在得到答案之前我先把我的仓库 Private 罢（https://github.com/SALTWOOD/SaltyQQChat）

EDIT: 看样子即使是通过 MCDR 引用还是要用 GPL 的，那我就闭源私用得了（抱歉打扰）

**Comment by AnzhiZhang** [comment, marked as answer]
我特此授权，阁下可以根据 [LGPL 3.0](https://www.gnu.org/licenses/lgpl-3.0.html) 协议使用 QQAPI 软件。


I hereby grant authorization for you to use the QQAPI software under the terms of the [LGPL 3.0](https://www.gnu.org/licenses/lgpl-3.0.html) license.
